### PR TITLE
fix(sasjs-web): fix service paths in generated web app HTML

### PR DESCRIPTION
**Pull request by krishna-acondy**
## Issue

https://github.com/sasjs/cli/issues/471

## Intent

Fix references to JS and CSS files in generated web app output.

## Implementation

* Added `/services` to file paths in tag hrefs.
* Made `streamServiceName` configurable from `streamConfig`.
* Fixed formatting on deploy logs.
* Showed web app URL on terminal after deployment.
* Removed noise from terminal output by changing `logger.info` to `logger.debug` where applicable (more left to do on this front).
* Removed more noise from terminal output by fetching the target once, and then reusing across compile, build and deploy steps.

## Checks

- [x] Code is formatted correctly (`npm run lint:fix`).
- [ ] Any new functionality has been unit tested - no new functionality.
- [x] All unit tests are passing (`npm test`).
- [x] All CI checks are green.
- [ ] JSDoc comments have been added or updated.
- [x] Reviewer is assigned.

### CODE REVIEW

**Review by saadjutt01 — APPROVED** [review]

File: src/commands/web/index.ts [hunk 1]
```diff
@@ -28,7 +28,8 @@ const exampleStreamConfig: StreamConfig = {
   streamWeb: true,
   streamWebFolder: '/example/folder/path',
   assetPaths: [],
-  webSourcePath: '/example/path'
+  webSourcePath: '/example/path',
+  streamServiceName: 'clickme'
 }
 
 export async function createWebAppServices(targetName: string) {
```
Comment (saadjutt01) [inline src/commands/web/index.ts:35]: Why not passing Target as passing to `compile` `build` `deploy`,

Comment (krishna-acondy) [reply src/commands/web/index.ts:35]: Good point. Fixing this now.

Comment (krishna-acondy) [reply src/commands/web/index.ts]: Fixed.

File: src/commands/deploy/deploy.ts [hunk 2]
```diff
@@ -177,12 +175,17 @@ async function deployToSasViya(
...
-      `Job execution completed! Log is available at ${path.join(
+    process.logger?.success(
+      `Deployment completed! Log is available at ${path.join(
         logFilePath,
         `${path.basename(deployScript).replace('.sas', '')}.log`
       )}`
     )
+
+    if (!!target.streamConfig?.streamWeb) {
+      const webAppStreamUrl = `${target.serverUrl}/SASJobExecution?_PROGRAM=${target.appLoc}/services/${target.streamConfig.streamServiceName}`
```
Comment (allanbowe) [inline src/commands/deploy/deploy.ts:186]: This won't work for SAS 9 - I think we have some logic for this in the adapter (SASJobExecution vs SASStoredProcess)

Comment (krishna-acondy) [reply src/commands/deploy/deploy.ts:186]: This line is in the `deployToSasViya` function.
But that's a valid point regardless - I've now added a similar log for SAS9 with the correct URL.

**Review by allanbowe — APPROVED** [review]

**Pull request merged**